resources/building-mobile-extensions/shared-states-and-events.md

```diff
@@ -3,19 +3,23 @@
 A shared state is composed of the following:
 
 * A name, which is the name of the extension or module that owns this shared state.
-* An event , which is an event that contains data that an extension wants to expose to other extensions or modules.
+* An event, which is an event that contains data that an extension wants to expose to other extensions or modules.
 
-**Important**: Every event does not result in an updated shared state. Shared states have to be specifically set, which causes events to be sent that the extensions and internal modules can listen for to be updated.
+{% hint style="warning" %}
+Not every event results in an updated shared state. Shared states have to be specifically set, which causes events to be sent that the extensions and internal modules can listen for to be updated.
+{% endhint %}
 
 ## Using a shared state
 
 Modules and extensions use events and shared states to communicate with each other. The events allow modules to be relatively decoupled, but shared states are necessary when you have a dependency on a module’s information.
 
-A module that receives an event for which it is listening, for example, when Analytics listens for Configuration changes, might trigger some asynchronous work in response to that event, for example, a network request. When the results of that work results in another event being dispatched, the second event that was dispatched is tied to the original event that started the cycle. This process allows downstream listeners to retrieve the same shared state with the same event data that caused the additional work to occur.
+A module that receives an event for which it is listening, for example, when Adobe Analytics listens for configuration changes, may trigger asynchronous work in response to that event. When the results of that work results in another event being dispatched, the second event that was dispatched is tied to the original event that started the cycle. This process allows downstream listeners to retrieve the same shared state with the same event data that caused the additional work to occur.
```

Comment:
Do we need "for example, a network request" still? Check with @emdobrin.

Reply:
... may trigger asynchronous work in response to that event. Examples of asynchronous work are a network request, processing the event data received or persisting some data on the disk. When the results of ...